src/storybook/stand-alone-documentaion/playground/Playground.stories.ts

```diff
@@ -5,4 +5,28 @@ export default {
   decorators: [withPlaygroundRenderer]
 };
 
-export const Playground = {};
+export const Playground = {
+  args: {
+    code: {
+      jsx: `<div style={{ display: "flex", flexDirection: "column", gap: "12px" }}>
```

Comment:
Maybe change it to have className with code in the css section that includes the content

Reply:
sure. I wanted to use the `Flex` component but we have some issues there with the static properties

Reply:
I'll look into it, not sure why it happens.
I guess it's an issue with react-jsx-parser library. I'll try to check how it renders with react-live's renderer